packages/web-components/tools/babel-plugin-create-react-custom-element-type.js

```diff
@@ -209,6 +210,69 @@ function createMetadataVisitor(api) {
       }
 
       const metadata = getPropertyMetadata(path);
+
+      // Checks if the current component extends a local dotcom component
+      if (context.parentDescriptorSource && !context.parentDescriptorSource.includes('carbon-web-components')) {
```

Comment:
```suggestion
      if (!context?.parentDescriptorSource?.includes('carbon-web-components')) {
```

would something like this work as well?

Reply:
ahh i thought it would too, but it then passes if a component does not have a `parentDescriptorSource`. gonna revert back